production/docker/config/loki-frontend-config.yaml

```diff
@@ -0,0 +1,42 @@
+auth_enabled: false
+
+http_prefix:
+
+server:
```

Comment:
I'd like suggest better ingester config:

```
ingester:
    chunk_block_size: 262144
    chunk_encoding: snappy
    chunk_idle_period: 2h
    chunk_retain_period: 6m
    chunk_target_size: 1.572864e+06
```

Reply:
I just realized you have 2 configs but you could have only one and use `-target=all` or `-target=query-frontend`